java/sage/FileDownloader.java

```diff
@@ -239,7 +241,7 @@ public synchronized Object downloadFile(String serverName, String srcFile, java.
           }
           break;
         }
-        fileOut = new FastRandomFile(myDestFile, "rw", Sage.I18N_CHARSET);
+        fileOut = new LocalSageFile(myDestFile, false);
```

Comment:
FastRandomFile does buffered writes, so you should probably wrap this in the buffering class for equivalence.